hphp/runtime/vm/jit/code-cache.cpp

```diff
@@ -97,7 +98,14 @@ CodeCache::CodeCache()
     exit(1);
```

Comment:
Ok, I can accept that.

I've found that instead of shrinking the sizes of the various caches to fit within 128MB, I can change the memory mapping to place ACold before AProf. This allows direct jumps from AHot to reach ACold, and keeps the significant gains for OSS-mediawiki (in conjunction with #7661). Would an ARM specific change to the cache's layout be acceptable?